components/authz-service/engine/opa/policy/introspection_v2.rego

```diff
@@ -26,24 +26,30 @@ pair_matches_action[[pol_id, statement_id, pair]] {
 	authz.action_matches(pair.action, role_action)
 }
 
+# This causes the has_member set to be generated and memoized once instead of
+# for each policy. Moreover it prevents the backtracking from generating duplicate
+# answers which be evaluated by the resource and action matchers.
```

Comment:
Not clear on the meaning due to the grammar glitch "which be"; not sure what it should read.

Reply:
Whoops, `s/which be/that have to be/`